aiohttp/web_reqrep.py

```diff
@@ -435,8 +445,12 @@ def enable_chunked_encoding(self, chunk_size=None):
         self._chunked = True
         self._chunk_size = chunk_size
 
-    def enable_compression(self, force=False):
-        """Enables response compression with `deflate` encoding."""
+    def enable_compression(self, force=None):
```

Comment:
I guess `force` may be `ContentCoding` enumeration for forcing compression to exact encoding.